crates/biome_rule_options/src/shared/sort_order.rs

```diff
@@ -0,0 +1,9 @@
+#[derive(
+    Clone, Copy, Debug, Default, Eq, PartialEq, serde::Deserialize, serde::Serialize, biome_deserialize_macros::Deserializable,
+)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+pub enum SortOrder {
```

Comment:
I think you should add:

```suggestion
#[serde(rename_all = "camelCase")]
pub enum SortOrder {
```

Reply:
Resolved